### PR TITLE
update plugin readmes with options

### PR DESCRIPTION
**Pull request by hzoo**
We should probably add an explanation for each option as well.

(And update the website/figure out how to just use the readme instead of copy pasting/syncing)

### CODE REVIEW

File: packages/babel-plugin-transform-es2015-arrow-functions/README.md [hunk 1]
```diff
@@ -14,10 +14,18 @@ $ npm install babel-plugin-transform-es2015-arrow-functions
...
+```js
+// without options
 {
   "plugins": ["transform-es2015-arrow-functions"]
 }
+
+// with options
+{
+  "plugins": [
+    ["transform-es2015-arrow-functions", { "spec": "true" }]
```
Comment (goto-bus-stop) [inline packages/babel-plugin-transform-es2015-arrow-functions/README.md]: Should this (and the other occurences of string `"true"`) be a plain boolean `true` instead?

Comment (hzoo) [inline packages/babel-plugin-transform-es2015-arrow-functions/README.md]: Oops - true!

**Comment by loganfsmyth** [comment]
Nice, looks good, this still seems like a great first step.

**Comment by hzoo** [comment]
Ok will probably need help with those since I probably don't know what all of them do.

**Pull request merged**